dev/README_RELEASE_PROVIDER_PACKAGES.md

```diff
@@ -176,6 +175,27 @@ breeze release-management prepare-provider-documentation \
  --base-branch provider-cncf-kubernetes/v4-4 cncf.kubernetes
 ```
 
+In case you want to **just** regenerate the documentation because you fixed something in the templates, add
+`--reapply-templates` flag to the command above. This refreshes content of:
+
+* `__init__.py` in provider's package
+* Provider Commits
+* Provider index for the documentation
+* Provider README file used when publishing package in PyPI
+
+If you want to just update min airflow version for all packages, you should modify `MIN_AIRFLOW_VERSION`
+in `dev/provider_packages/prepare_provider_packages.py` and run the `prepare-provider-documentation`
+command with `--only-min-version-update` flag. this will only update the min version in
+the `__init__.py` files and package documentation without bumping the provider versions.
```

Comment:
```suggestion
If you want to just update the min airflow version for all packages, you should modify `MIN_AIRFLOW_VERSION`
in `dev/provider_packages/prepare_provider_packages.py` and run the `prepare-provider-documentation`
command with the `--only-min-version-update` flag. This will only update the min version in
the `__init__.py` files and package documentation without bumping the provider versions.
```